File: CS50x/mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height;

    do
    {
        height = get_int("Height: ");
    }
    while (height < 1 || height > 8);

    for (int linha = 1; linha < height + 1; linha++)
    {
        for (int j = 0; j < height - linha; j++)
        {
            printf(" ");
        }

        for (int j = 0; j < linha; j++)
        {
            printf("#");
        }

        printf("  ");

        for (int j = 0; j < linha; j++)
        {
            printf("#");
        }

        printf("\n");
    }
}

File: CS50x/sentimental-mario-less/mario.py
height = 0

while True:
    height = input("Height: ")
    if height.isdigit():
        height = int(height)
        if height > 0 and height < 9:
            break

#    except ValueError:
#    else:

for linha in range(1, height+1):
    for j in range(height-linha):
        print (' ',end="")
    for j in range(linha):
        print ("#",end="")
    print ("")
